grasp_generation/utils/isaac_validator.py

```diff
@@ -448,6 +485,29 @@ def run_sim(self):
             # Step physics if not paused
             if not self.is_paused:
                 gym.simulate(self.sim)
+
+                if self.camera_handles and sim_step_idx > 0:
+                    gym.step_graphics(self.sim)
+                    gym.render_all_camera_sensors(self.sim)
+                    for ii, env in enumerate(self.camera_envs):
+                        self.video_frames[ii].append(
+                            gym.get_camera_image(
+                                self.sim,
+                                env,
+                                self.camera_handles[ii],
+                                gymapi.IMAGE_COLOR,
+                            ).reshape(
+                                int(
```

Comment:
Nit: I think it's slightly cleaner to store self.camera_properties so you can use self.camera_properties.width, rather than using int(CAMERA_IMG_HEIGHT / RESOLUTION_REDUCTION_FACTOR_TO_SAVE_SPACE)